testservice/service.go

```diff
@@ -246,7 +249,7 @@ func main() {
 	loggers := ldlog.NewDefaultLoggers()
 	loggers.SetMinLevel(logLevelFromName(os.Getenv("LD_LOG_LEVEL")))
 
-	port := "8000"
+	port := "10000"
```

Comment:
Was changing the dynamo port a no-go, rather than changing the test service's port?

Reply:
I could have changed the dynamodb port. But since every other SDK is going to have to override it (and then each of us will when doing local work), I thought it might be better to try and get those ports on the standard ones, and then maybe we can align on an SDK test harness port.